handlers/leagues.js

```diff
@@ -43,7 +45,7 @@ var onLeaguesInMonthResponse = function onLeaguesInMonthResponse(message, callba
 
     if (response.eresult === 1) {
         if (this.debug) util.log("Received leagues in month response " + response.eresult);
-        this.emit("leaguesInMonthData", response.eresult, response);
+        this.emit("leaguesInMonthData", response.month, response.year, response.leagues);
```

Comment:
I don't know actually, this isn't the only  place where we do this actually, I think in the majority of cases we filter the result out, that's why I modified it here as well. It's kind off a theme throughout the handlers that, if you want the error code, you need to use the callback function. I don't really see the added value of returning a value which will always be 1...